app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -410,7 +410,7 @@ def traces_helpers(search:)
         ::Katello::HostTracer.helpers_for(traces)
       end
 
-      def package_names_for_job_template(action:, search:)
+      def package_names_for_job_template(action:, search:, versions:)
```

Comment:
```suggestion
      def package_names_for_job_template(action:, search:, versions: nil)
```
This method is called during install and remove as well, so `versions` should not be a required argument. Adding `nil` here will make it an optional arguments; I think this should fix my error.

Reply:
(however, now that it's optional you'll have to account for that in the `upgrade` case :)